requirements.txt

```diff
@@ -43,7 +43,7 @@ pympler>=0.6
 pyopenssl==17.5.0
 python-coveralls>=2.9.1
 pyyaml>=3.13,<=5.1
-Quart>=0.12.0
+Quart>=0.6.15
```

Comment:
This is the latest version of Quart that you can install on Centos 8 with the default python3 package. I haven't noticed any side effects related to this downdrade.